modules/deleting-machine-pools-cli.adoc

```diff
@@ -7,6 +7,11 @@
 = Deleting a machine pool using the ROSA CLI
 You can delete a machine pool for your Red Hat OpenShift Service on AWS (ROSA) cluster by using the ROSA CLI.
 
+[NOTE]
+====
+For users of ROSA CLI `rosa` version 1.2.25 and prior versions, the machine pool (ID='Default') created along with the cluster cannot be deleted. For users of ROSA CLI `rosa` version 1.2.26 and beyond, the machine pool (ID='worker') created along with the cluster can be deleted as long as there is one machine pool within the cluster that contains no taints and at least two replicas for a single-AZ cluster or three replicas for a multi-AZ cluster.
```

Comment:
```suggestion
For users of ROSA CLI `rosa` version 1.2.25 and earlier versions, the machine pool (ID='Default') that is created along with the cluster cannot be deleted. For users of ROSA CLI `rosa` version 1.2.26 and later, the machine pool (ID='worker') that is created along with the cluster can be deleted as long as there is one machine pool within the cluster that contains no taints, and at least two replicas for a single-AZ cluster or three replicas for a multi-AZ cluster.
```
Does 'AZ' modify 'cluster'? Should it be 'single AZ-cluster' and 'multi AZ-cluster'?